_non-cosmos/namada/chain.json

```diff
@@ -0,0 +1,154 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "namada",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://namada.net",
+  "pretty_name": "Namada",
+  "chain_type": "namada",
```

Comment:
chain_type "namada" is currently not acceptable. We'll either need to change to "unknown" or add "namada" to the chain.schema.json file.

Reply:
@JeremyParish69 what is you recommended solution here?

Reply:
I suppose since there isn't any plan for a chain to re-use "namada" as a chain_type, and no special validation exceptions are needed, I'd recommend chain_type "unknown", at least for now.